services/statestorage/test/_stress_test.go

```diff
@@ -0,0 +1,79 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/test"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"runtime"
+	"testing"
+	"time"
+)
+
+const TpxAvg = 100
+const TxCount6Months100Tps int = 6 * 30 * 24 * 60 * 60 * TpxAvg
+const BlockSizeMax = 200
+const Users = 1000000
+
+
+func TestSimulateStateInitFlowForSixMonthsAt100Tps(t *testing.T) {
+	test.WithContext(func(ctx context.Context) {
+		d := newStateStorageDriver(1)
+
+		// generate User keys
+		userKeys := make([][]byte, Users)
+		for i := range userKeys {
+			userKeys[i] = make([]byte, 32)
+			rand.Read(userKeys[i])
+		}
+
+		var txCount int
+		var commits int
+		var createInputDuration time.Duration
+		var commitDuration time.Duration
+		start := time.Now()
+		tickStart := start
+		for txCount < TxCount6Months100Tps {
+			// create input for current simulated block
+			generationStart := time.Now()
+			commitTxs, commit := getRandomCommit(userKeys)
+
+			createInputDuration += time.Now().Sub(generationStart)
+			txCount += commitTxs
+
+			// commit state
+			d.commitStateDiff(ctx, commit)
+
+			// print progress every 100000 commits
+			if commits++; commits % 100000 == 0 {
+				ms := runtime.MemStats{}
+				runtime.ReadMemStats(&ms)
+				elapsedTick := time.Now().Sub(tickStart)
+				elapsed := time.Now().Sub(start)
+				commitDuration = elapsed - createInputDuration
+				fmt.Printf("tick: %v, commit: %v, elapsed: %v, progress: %d, HeapSys: %dMB, HeapAlloc: %dMB\n", elapsedTick, commitDuration, elapsed, 100*txCount/TxCount6Months100Tps, ms.HeapSys/ (1024 * 1024), ms.HeapAlloc/ (1024 * 1024))
```

Comment:
Consider `t.Log` / `t.Logf` instead of `fmt.Printf`. If the system has its own stdout then these outputs are hard to find whereas `t.Log` is designed for this purpose

Reply:
In the summary log I changed to `t.Logf` but in this line I left in as is because t.Logf does not print until the test is done and this is a progress prompt